js/rank_states_utils.js

```diff
@@ -191,7 +200,12 @@ function rankEm(barData) {
           .attr('class',null)
           .transition().duration(600)
             .style('opacity', 0);
-        
+        gtag('event', 'dropped state correctly' , {
+          'event_category': 'figure',
+          'event_label': draggedState,
+          'sessionId': sessionId,
+          'timestamp': timestamp
+        });
         // is this the last one? if so, remove the directions    
         if (svgStates.select('#ranked-states-bars').selectAll('rect').filter('*:not(.locked-rank-bar)').empty()){
```

Comment:
wondering if we should stick an event in here - 'completed game' or something?

Reply:
We could, although we should be able to determine that based on number of correct state drops.

Reply:
is that easy to summarize in the analytics data that comes back for each user? So it would be simple to know how many users completed the game?

Reply:
Hmm, simple in R with a little dplyr, although thinking about it now, it might be good to be able to see that quickly through the GA UI.  I will add it.  I was trying to not get too out of hand with logging events.

Reply:
Yeah, that makes sense. Just wanted to double check that we are making it easy for ourselves to get the information we want! I will merge once the new code pops up here.